src/content/release-notes.mdx

```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.20
+
+21 May 2024
+
+This version is compatible with Kubernetes versions 1.26 to 1.29
+
+### New Features
+
+- Add support for Openshift: Openshift Routes now show as Okteto Endpoints  <!-- 8025 -->
+- Enable Pull Secrets by default  <!-- 8001 -->
+- Added onboarding guides to our documentation on [how to get started with Okteto Insights](admin/okteto-insights.mdx) <!-- 729 -->
+
+### Improvements
+
+- Namespaces without an owner will now be picked up by the Garbage Collector  <!-- 8018 -->
+- Added the ability to assign a PriorityClass to Okteto components  <!-- 7966 -->
+- Added custom tolerations to the Helm Chart and standardize their helm values  <!-- 8029 -->
```

Comment:
We are adding `globals` tolerations that apply to any Okteto component (pods in the okteto namespace)